### DISCUSSION TITLE
Student-t family

### DISCUSSION THREAD

**Discussion by Nathan-Hebert** [sdmTMB/sdmTMB#300] [Q&A] [answered]
Is it inadvisable to fit a model using the Student-t family with df < 3? If I'm not mistaken, the t-distribution variance is undefined for 0<df≤1 and infinite for 1<df≤2. Although maybe these details aren't relevant, as we're talking about a scaled and shifted t-distribution?

**Comment by seananderson** [comment, marked as answer]
This is probably a more general stats question than sdmTMB specific. I'm not sure. My guess is you can get away with df = 2 or at least very close to 2 given the parameter is not estimated in sdmTMB. Below that I'm not sure. If you want to find out, I would do some basic simulation testing. Start with random fields off for simplicity. If it looks promising under a simple example you can try simulating data that resembles your full dataset of interest and fitting that.

**Reply by Nathan-Hebert** [reply]
Thank you! :)
